### DISCUSSION TITLE
请教下关于“隐藏窗口到托盘”

### DISCUSSION THREAD

**Discussion by cc19264997** [shuax/MouseInc.Settings#282] [Q&A] [answered]
其他设置-全局热键-隐藏窗口到托盘的快捷键是CTRL+ALT+Z，我想把它更改为CTRL+· （就是数字1旁边的·键），设置完以后是无效的。不知道该怎么办。希望耍下大佬能帮帮我！谢谢

**Comment by cc19264997** [comment]
再一个就是全局热键能否设置排除某些程序？嘻嘻~（脑补表情包）

**Comment by shuax** [comment, marked as answer]
那个键不是·啊，你直接按就是`，不要开中文输入法。全局热键目前是全局的。

**Reply by cc19264997** [reply]
原来是我开了输入法。多谢啊！
